### ISSUE TITLE
wrtc-star1.par.dwebops.pub pubsub stopped working

### ISSUE THREAD

**Issue by StationedInTheField** [libp2p/js-libp2p-webrtc-star#258]
Hey, we're using `wrtc-star1.par.dwebops.pub` for pubsub in our example and it broke yesterday. The other address works but on this address peers listening to the same topic can't see each other

**Comment by vasco-santos** [comment]
Hey @StationedInTheField 
Do peers connect via it, or they fail to connect?

**Comment by vasco-santos** [comment]
I just tried out the `wrtc-star1.par.dwebops.pub` and the connectivity seems fine. Can you give me more details?

**Comment by StationedInTheField** [comment]
They connect, 10 swarm peers listed and ``Swarm listening on /dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star/p2p/QmXNudapyb5pxgEC9WALkaeohb8gC8pmKw9Nr4HApgpCDV`` but the peers can't see each other on the same topic. If I go and change it to the address here: https://github.com/QuestNetwork/qDesk/blob/0.9.4/src/app/swarm.json it works again.

**Comment by vasco-santos** [comment]
Maybe it was some availability issue. Have you tried again? I don't see any reason for only one of the public star servers to work.
Just to be clear, are you connection all the peers to that single star server for trying it? 
One last observation, you should run your own star server at some point and do not rely on the public ones as there availability is not guaranteed and are shared by a lot of people experimenting.

**Comment by StationedInTheField** [comment]
It's really strange and it still isn't working while on the other address it works. 

Yeah of course. We are running our own star node and recommend every business that uses it to run their own that's why we have the editable list in Settings > IPFS. But this is our example app, so developers can test it. I'm implementing some additional peering, so the app always has or at least can request a list of working star nodes.

**Comment by vasco-santos** [comment]
> It's really strange and it still isn't working while on the other address it works.

What I was trying to understand is if both pubsub subscriber peers are using the same star server. I will make some experiments meanwhile

**Comment by StationedInTheField** [comment]
yes, they're the same app with the same configuration, just different repo names and user data - and like i said works fine with the sjc address

**Comment by vasco-santos** [comment]
I could not replicate this. If you can get any extra information, or a replication test, it would be super helpful

**Comment by vasco-santos** [comment]
I am closing this issue! Feel free to re-open if this is still an issue

**Comment by StationedInTheField** [comment]
Well you can test it with the Web Demo, it also happen with the sjc star configured now: 

https://gateway.pinata.cloud/ipfs/QmZNvMbTPCEW1vebsEc5L2dx18PeYCKgwP5oMCVREkmr3p/

On chat in the left bar via the context menu on the + icon you can create a channel, then on the right bar in settings, you can create an invite, in a second browser window you can then paste the invite via "join channel" in the context menu on the + icon on the left.

The console logs are very verbose and whenever a message in a channel is received there's output from pubsub.subscribe and with both of those windows you can see that messages sent in window a (sayHi would be the one to ask to join) are not received in window b. If I however use one of our stars, it works with no issues just like it did on the par initially, and then on the sjc for a while.

Relevant code for the pubsub is in: https://github.com/QuestNetwork/quest-pubsub-js - But it's just simple subscribe and publish.

**Comment by vasco-santos** [comment]
I could replicate now via the `libp2p in the browser` example that the peers do not get discovered+connected using `/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star`. The peers are able to connect to the signal server, and emit the `ss-join` message. However, it seems that the server is not getting it correctly. The other star server is working perfectly.

@mburns Can you provide us logs from that server?

cc @jacobheun

**Comment by mburns** [comment]
Looks like an old image got deployed (`libp2p/websocket-star-rendezvous:v0.3.0`).

I've stopped that and got `libp2p/js-libp2p-webrtc-star` running again, which matches `wrtc-star2`.

**Comment by vasco-santos** [comment]
Thanks @mburns this is now working on the example.
@StationedInTheField can you let me know if this is sorted out on your end?
